spec/features/authorization_of_conversation.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+feature "user visits a conversation show page" do
+  context "unauthorized user" do
+    let(:bad_user) { FactoryGirl.create(:user) }
+    let!(:conversation) { FactoryGirl.create(:conversation) }
+
+    before do
+      visit root_path
+      click_link "Sign In"
+      fill_in 'Email', with: bad_user.email
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.